fitbenchmarking/fitting_benchmarking.py

```diff
@@ -516,3 +517,22 @@ def empty_contents_of_folder(results_dir):
             os.unlink(file_path)
         elif os.path.isdir(file_path):
             shutil.rmtree(file_path)
+
+
+def setup_results_directory(results_dir):
```

Comment:
please add documentation for this method

Reply:
I am often skipping documentation as I will add it all in #48 , but can add it now if that's preferred.

Reply:
Yes please add as you add new functions since this is point in time where it is most fresh in your mind what the method is doing